exercises/resistor-color/test/resistor_color_test.dart

```diff
@@ -1,30 +1,31 @@
 import 'package:resistor_color/resistor_color.dart';
 import 'package:test/test.dart';
 
-void main() {
-  final resistorColor = new ResistorColor();
+final resistorColor = ResistorColor();
 
+void main() {
   group('ResistorColor', () {
     group('Color codes', () {
-      test("Black", () {
-        final int result = resistorColor.colorCode("black");
+      test('Black', () {
+        final int result = resistorColor.colorCode('black');
         expect(result, equals(0));
       }, skip: false);
 
-      test("White", () {
-        final int result = resistorColor.colorCode("white");
+      test('White', () {
+        final int result = resistorColor.colorCode('white');
         expect(result, equals(9));
       }, skip: true);
 
-      test("Orange", () {
-        final int result = resistorColor.colorCode("orange");
+      test('Orange', () {
+        final int result = resistorColor.colorCode('orange');
         expect(result, equals(3));
       }, skip: true);
     });
 
-    test("Colors", () {
+    test('Colors', () {
       final List<String> result = resistorColor.colors;
-      expect(result, equals(["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]));
+      expect(result,
+          equals(<String>['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white']));
```

Comment:
Add a trailing comma to the list to give it a better formatting.
Alternatively add a trailing comma after second argument of `expect` so both arguments can be on their own line, which might produce a better formatting
```suggestion
          equals(<String>['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white',]));
```
OR
```suggestion
          equals(<String>['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white']),);
```

Reply:
Thanks, I'll take the former suggestion :)